docker-compose.yml

```diff
@@ -82,19 +82,16 @@ services:
     command: gunicorn --reload -b 0.0.0.0:8000 warehouse.wsgi:application
     env_file: dev/environment
     volumes:
-      # We specify all of these directories instead of just . because we want to
-      # avoid having ./node_modules from the host OS being shared with the docker
-      # container, and since there's no way to exclude a directory, the only way
-      # to make this work is to share multiple, smaller directories. These cover
-      # the important things that we want to share, but changes to requirements
-      # or any file not in these directories will require a rebuild.
+      # We specify paths explicitly to only add what we need, and to avoid
+      # adding the entire repository.
       # The :z option fixes permission issues with SELinux by setting a
       # permissive security context.
       - ./dev:/opt/warehouse/src/dev:z
       - ./docs:/opt/warehouse/src/docs:z
       - ./warehouse:/opt/warehouse/src/warehouse:z
       - ./tests:/opt/warehouse/src/tests:z
       - ./htmlcov:/opt/warehouse/src/htmlcov:z
+      - ./pyproject.toml:/opt/warehouse/src/pyproject.toml:z
```

Comment:
Did you confirm that this works? I've seen issues when the path specified isn't absolute Ala `${PWD}/pyproject.toml`

Reply:
Yes - confirmed - the leading `./` supplies the path (see the next entry for `.coveragerc`

Reply:
cool, thanks. that might just be old memories.